Documentation/Flashing.md

```diff
@@ -66,7 +66,7 @@ Example, to be run as root, once the unit has been plugged in DFU mode and auto-
 ```bash
 FW=ts100.hex
 unset NAME
-eval $(lsblk -P -p -d --output NAME,MODEL|grep "DFU Disk")
+eval $(lsblk -P -p -d --output NAME,MODEL|grep "DFU[ _]Disk")

```

Comment:
On my laptop, the TS100 shows up as "DFU_Disk" instead of "DFU Disk"". I dunno if this is different on other distros/irons, but this now matches both.

Reply:
The underscore is dependent on the DFU version